battleship.py

```diff
@@ -2,6 +2,44 @@
 
 import random
 
+import pygame
+
+pygame.mixer.init()
+
+
+# sounds
+
+
+def hit_music():
+    pygame.mixer.music.load('hardcore.mp3')
+    pygame.mixer.music.play()
+
+
+def miss_music():
+    pygame.mixer.music.load('music.mp3')
+    pygame.mixer.music.play()
+
+
+def skip_music():
+    pygame.mixer.music.load('fricks.mp3')
+    pygame.mixer.music.play(1)
+
+
+def hit_sound():
+    sound = pygame.mixer.Sound('388528__eflexmusic__artillery-explosion-close-mixed.wav')
+    sound.play()
+
+
+def miss_sound():
+    sound = pygame.mixer.Sound('9508__petenice__splash.wav')
```

Comment:
We should rename the files so they don't look ugly in the code. Maybe one word like "splash.wav"